INSTALL.md

```diff
@@ -0,0 +1,105 @@
+# Installation
+
+## Table of contents
+
+- [Arch-based distributions](#arch-based-distributions)
+- [Debian](#debian)
+- [Fedora](#fedora)
+- [OpenMandriva](#openmandriva)
+- [openSUSE](#opensuse)
+- [Pop!_OS](#pop_os)
+- [Ubuntu-based distributions](#ubuntu-based-distributions)
+- [AppImage](#appimage)
+- [Flatpak](#flatpak)
+- [Prebuilt releases](#prebuilt-releases)
+- [Source](#source)
+	- [Building](#building)
+	- [Installing](#installing)
+
+## Arch-based distributions
+[`gamehub-git`](https://aur.archlinux.org/packages/gamehub-git/) and [`gamehub`](https://aur.archlinux.org/packages/gamehub/) are available in AUR.
+
+## Debian
+Unfortunately, there is no package available in the Debian repository, so it will be required to run the [`/scripts/build.sh build_deb`](../scripts/build.sh#L171-L210) script. The script will install the dependencies and GameHub.
+
+**Build dependencies**
+* `meson`
+* `valac`
+* `libgtk-3-dev`
+* `libglib2.0-dev`
+* `libwebkit2gtk-4.0-dev`
+* `libjson-glib-dev`
+* `libgee-0.8-dev`
+* `libsoup2.4-dev`
+* `libsqlite3-dev`
+* `libxml2-dev`
+* `libpolkit-gobject-1-dev`
+* `libunity-dev` (optional, required for launcher icon quicklist, progress indicator and counter; pass `-Duse_libunity=true` to `meson` to use)
+* `libmanette-0.2-dev`, `libx11-dev`, `libxtst-dev` (optional, required for gamepad support)
+
+```bash
+git clone https://github.com/tkashkin/GameHub.git
+cd GameHub
+scripts/build.sh build_deb
+```
+
+## Fedora
+```bash
+sudo dnf install gamehub
+```
+
+## OpenMandriva
+[`gamehub`](https://abf.openmandriva.org/openmandriva/gamehub/build_lists) is available in the OpenMandriva repository.
+
+## openSUSE
+```bash
+sudo zypper install gamehub
+```
+
+## Pop!_OS
+
+```bash
+sudo apt install com.github.tkashkin.gamehub
+```
+
+## Ubuntu-based distributions
+Install Debian package from the [releases page](https://github.com/tkashkin/GameHub/releases) or import the [PPA](https://launchpad.net/~tkashkin/+archive/ubuntu/gamehub):
+
+**THIS EXCLUDES [POP!_OS](#pop_os)!**
+```bash
+# install if `add-apt-repository` is not available
+sudo apt install --no-install-recommends software-properties-common
+
+sudo add-apt-repository ppa:tkashkin/gamehub
+sudo apt update
+sudo apt install com.github.tkashkin.gamehub
+```
+
+## AppImage
+AppImages can be found in the [releases page](https://github.com/tkashkin/GameHub/releases).
+
+## Flatpak
+Unfortunately, there is no package available in the Flathub repository, so it will be required to run the [`/scripts/build.sh build_flatpak`](../scripts/build.sh#L311-L334) script:
```

Comment:
There are prebuilt flatpaks on releases page, `flatpak install GameHub-*.flatpak` should be enough.

Reply:
you're right, i wasn't aware of that, i'll work on it

Reply:
also, i would suggest you to add the Flatpak in Flathub: https://github.com/flathub/flathub/

Reply:
It was rejected from flathub, but that was a long time ago.

I may try to add it again in the future, but currently there are problems with flatpaks and I would recommend to use native distro packages or build GameHub from source.

Reply:
Done https://github.com/tkashkin/GameHub/pull/407/commits/bab266e074b2045d904b0e98f661408efa65c487

Reply:
> It was rejected from flathub, but that was a long time ago.

ouch...

> I may try to add it again in the future, but currently there are problems with flatpaks and I would recommend to use native distro packages or build GameHub from source.

sure, but should we add a warning?